buildAndReleaseTask/task.json

```diff
@@ -13,17 +13,16 @@
     "version": {
         "Major": 1,
         "Minor": 0,
-        "Patch": 7
+        "Patch": 16
```

Comment:
This is the current patch version in the marketplace. Also since the added scope will require explicit user-authorization (see screenshot in the PR description) I will defer the decision of whether or not the major or minor version should be bumped, to you.

Reply:
I'd suggest a major version bump on this basis:
1. There's little harm in bumping the major version. 
2. Since this change requires new permissions, we would like our users to be able to continue using the pipeline task even if they decide they don't want to grant the new permissions. Users who pin to the minor version will experience an interruption in service if we release this as a minor version.